### ISSUE TITLE
Feature request: Larger SQL editor

### ISSUE THREAD

**Issue by ryanhamilton** [timestored/pulseui#116]
user-yx would like a larger SQL code editor.

```
I have a question on the dashboard interface: I’m wondering if it is possible to make the editor panel (the panel on the top right corner in the screenshot) larger? As code gets longer it becomes quite difficult to see everything in the small panel.
```

They had 70+ lines of SQL code which was impossible to read.
I demonstrated the workaround of using the SQL editor page for editing.

**Comment by ryanhamilton** [comment]
user-ag (10) - User wants to expand the whole editor to full screen while editing.


![image](https://github.com/timestored/pulseui/assets/7273208/bea4ed94-01b7-4463-9174-233be03ae869)

**Comment by ryanhamilton** [comment]
+user-ll " seems I cannot resize the editor on the right hand side, and not sure if is my version issue if I break the q query in to multi lines the query would no more work. The need here is a resizebale editor for long q query editing "

**Comment by ryanhamilton** [comment]
+ user-di

**Comment by ryanhamilton** [comment]
user-ar user-sid

`Allow extension to the left of the coding panel, code is often too long and makes it hard to manage`

**Comment by ryanhamilton** [comment]
user-ar user-sid

> - Depending on the chart, when code is too long, the frame gets cropped vertically and we can't scroll. You can try to replicate by placing huge code in the right panel, selecting grid as the grid has params you can scroll, but if you select Sankey for instance you can't see the bottom of your code as there is no parameterization available (aka no frame below the code frame).

![image](https://github.com/user-attachments/assets/a3977620-1958-430f-abd2-a9a0f145bd91)

**Comment by ryanhamilton** [comment]
![image](https://github.com/user-attachments/assets/ec8d3aff-b827-4bc9-b7b8-138643c12fd7)

**Comment by ryanhamilton** [comment]
Released 3.04
